grammar.js

```diff
@@ -743,17 +746,9 @@ module.exports = grammar({
     )),
 
     pipe_expression: $ => prec.left(seq(
-      $.primary_expression,
+      $.expression,
       choice('->', '|>'),
-      choice(
-        $.value_identifier,
-        $.value_identifier_path,
-        $.variant_identifier,
-        $.polyvar_identifier,
-        $.nested_variant_identifier,
-        $.parenthesized_expression,
-        $.block,
-      ),
+      $.expression,
```

Comment:
This change makes `pipe_expression` rule equal to `binary_expression`, We can do a simplification here.

Reply:
See #177